# This file is part of HiGFXback

# requires
REQUIRES="autotools-wrappers-am-build"

pkg-config --exists --print-errors $REQUIRES || exit 1

# configure (ac-2.71; am-1.16)
aclocal -I m4; autoheader; autoconf; automake -a -c
./configure --prefix=

# build
make

# install
install -d $DESTDIR/bin
install src/cmp $DESTDIR/bin
install src/diff $DESTDIR/bin
install src/diff3 $DESTDIR/bin
install src/sdiff $DESTDIR/bin

# build.pc
install -d $DESTDIR/share/pkgconfig
cat > $DESTDIR/share/pkgconfig/diffutils-build.pc << EOF
Name: diffutils
Version: 3.8
Description: GNU utilities for comparing and merging files
Requires: $REQUIRES

exec=\\
/bin/cmp \\
/bin/diff \\
/bin/diff3 \\
/bin/sdiff
EOF

exit

# patch
--- diffutils-3.8.orig/configure.ac
+++ diffutils-3.8/configure.ac
@@ -30,6 +30,8 @@
                        color-tests parallel-tests])
 AM_SILENT_RULES([yes]) # make --enable-silent-rules the default.
 
+AM_MAINTAINER_MODE
+
 AC_CONFIG_HEADERS([lib/config.h:lib/config.hin])
 
 AC_PROG_AWK

# source
https://ftp.gnu.org/gnu/diffutils/diffutils-3.8.tar.xz
